scripts/add-cluster.sh

```diff
@@ -8,18 +8,23 @@ user_help () {
     echo "-hn, --host-ns        namespace where host-operator is running"
     echo "-s,  --single-cluster running both operators on single cluster"
     echo "-m,  --minishift      in case of using minishift"
+    echo "-kc,  --kube-config   kubeconfig for managing multiple clusters"
     exit 0
 }
 
 login_to_cluster() {
     if [[ ${SINGLE_CLUSTER} != "true" ]]; then
         if [[ ${MINISHIFT} != "true" ]]; then
+          if [[ -z ${KUBECONFIG} ]]; then
             echo "Write the server url for the cluster type: $1:"
             read -p '> ' CLUSTER_URL
             echo "Provide the token to log in to the cluster ${CLUSTER_URL}:"
             read -sp '> ' CLUSTER_TOKEN
             echo ""
             oc login --token=${CLUSTER_TOKEN} --server=${CLUSTER_URL}
+          else
+            oc config use-context "$1-admin"
```

Comment:
Please, add also a description to the readme how to set the contexts so they are properly used by the script - what are the expected names, etc...

Reply:
Addressed in  d4f3d0d